### PR TITLE
Add new groovy scripts for maven and neo4j

### PR DESCRIPTION
**Pull request by vaibhavsood**

### CODE REVIEW

**Comment by vaibhavsood** [comment]
Hi Tianon, wanted to check if can get some review for this, just thought would do a gentle reminder :smile:

File: dsl/docker-multiarch/images/maven.groovy [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+for (arch in multiarch.allArches()) {
+	meta = multiarch.meta(getClass(), arch)
+
+	freeStyleJob(meta.name) {
+		description(meta.description)
+		logRotator { daysToKeep(30) }
+		label(meta.label)
+		scm {
+			git {
+				remote { url('https://github.com/carlossg/docker-maven') }
```
Comment (tianon) [inline dsl/docker-multiarch/images/maven.groovy]: Missing `.git` here

File: dsl/docker-multiarch/images/maven.groovy [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			}
+		}
+		triggers {
+			upstream("docker-${arch}-openjdk", 'UNSTABLE')
+			scm('H H/6 * * *')
+		}
+		wrappers { colorizeOutput() }
+		steps {
+			shell(multiarch.templateArgs(meta) + '''
+sudo rm -rf tests
```
Comment (tianon) [inline dsl/docker-multiarch/images/maven.groovy]: This doesn't need `sudo`, does it?

File: dsl/docker-multiarch/images/neo4j.groovy [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+for (arch in multiarch.allArches()) {
+	meta = multiarch.meta(getClass(), arch)
+
+	freeStyleJob(meta.name) {
+		description(meta.description)
+		logRotator { daysToKeep(30) }
+		label(meta.label)
+		scm {
+			git {
+				remote { url('https://github.com/neo4j/docker-neo4j') }
```
Comment (tianon) [inline dsl/docker-multiarch/images/neo4j.groovy]: Missing `.git`

**Comment by tianon** [comment]
Generally OK here too! :+1:

File: dsl/docker-multiarch/images/maven.groovy (hunk 1 above)
Comment (vaibhavsood) [inline dsl/docker-multiarch/images/maven.groovy]: Corrected this

File: dsl/docker-multiarch/images/maven.groovy (hunk 2 above)
Comment (vaibhavsood) [inline dsl/docker-multiarch/images/maven.groovy]: It doesnt :smile:

File: dsl/docker-multiarch/images/neo4j.groovy (hunk 3 above)
Comment (vaibhavsood) [inline dsl/docker-multiarch/images/neo4j.groovy]: Corrected this

**Comment by vaibhavsood** [comment]
Have made the changes, thanks!

**Comment by tianon** [comment]
:+1:

**Pull request merged**